GOVERNANCE.md

```diff
@@ -93,8 +93,12 @@ Council Members will have the responsibility of
 * Make decisions when regular community discussion doesn’t produce consensus on an issue in a reasonable time frame.
 * Make decisions about strategic collaborations with other organizations or individuals.
 * Make decisions about the overall scope, vision and direction of the project.
+* Developing funding sources
+* Deciding how to disburse funds
```

Comment:
Should we add something about the council needing to consult the core developers (basically my comment on slack)?

Reply:
Added